driver/src/main/java/org/neo4j/driver/v1/Entry.java

```diff
@@ -21,11 +21,12 @@
 /**
  * Immutable pair of a key and a value
  *
- * @see PropertyMapAccessor
+ * @see MapAccessor
+ * @see RecordAccessor
  * @param <V> the Java type of the contained value
  */
 @Immutable
-public interface Property<V>
+public interface Entry<V>
```

Comment:
Maybe.. I usually don't import it and just use Map.Entry directly in my Java code; that was the thinking here but I realize now that is not a good thing to assume by default. What would be a good generic alternative? We could just have a type `Pair<K, V>`? Wdyt?